docs/frameworks.js

```diff
@@ -0,0 +1,124 @@
+module.exports = {
+  frameworks: ['react', 'react-ts', 'vue', 'angular'],
+  features: [
+    {
+      name: 'a11y',
+      unsupported: [],
+    },
+    {
+      name: 'Actions',
+      unsupported: [],
+      path: 'essentials/actions',
+    },
+    {
+      name: 'Backgrounds',
+      unsupported: [],
+      path: 'essentials/backgrounds',
+    },
+    {
+      name: 'cssresources',
+      unsupported: [],
+    },
+    {
+      name: 'design-assets',
+      unsupported: [],
+    },
+    {
+      name: 'Docs',
+      unsupported: [],
+      path: 'writing-docs/introduction',
+    },
+    {
+      name: 'events',
+      unsupported: ['svelte', 'riot'],
+    },
+    {
+      name: 'google-analytics',
+      unsupported: [],
+    },
+    {
+      name: 'graphql',
+      supported: ['react', 'angular'],
+    },
+    {
+      name: 'jest',
+      unsupported: [],
+    },
+    {
+      name: 'knobs',
+      unsupported: [],
+    },
+    {
+      name: 'links',
+      unsupported: ['marko'],
```

Comment:
Looks suspicious to me. Was this copied from the existing table or did you make it up?

Reply:
Copied from existing table